class.jetpack.php

```diff
@@ -468,6 +468,12 @@ private function __construct() {
 		add_action( 'init', array( $this, 'deprecated_hooks' ) );
 
 		/*
+		 * Enable enhanced handling of previewing sites in Calypso
+		 */
+		require_once JETPACK__PLUGIN_DIR . '_inc/lib/class.jetpack-iframe-embed.php';
+		add_action( 'init', array( 'Jetpack_Iframe_Embed', 'init' ), 9, 0 );
```

Comment:
Let's move this action to the file so everything is in a single place.

Reply:
I wrapped the class sourcing & init hook in 1a6aea2

Reply:
> Let's move this action to the file so everything is in a single place.

I tend to keep classes in their own files, myself so sourcing them is distinct from executing any code. @eliorivero, is this a hard requirement here?